src/main/java/nextstep/optional/Users.java

```diff
@@ -13,11 +15,9 @@ public class Users {
             new User("honux", 45));
 
     User getUser(String name) {
-        for (User user : users) {
-            if (user.matchName(name)) {
-                return user;
-            }
-        }
-        return DEFAULT_USER;
+        return users.stream()
+                .filter(user1 -> user1.matchName(name))
```

Comment:
```suggestion
                .filter(user -> user.matchName(name))
```
user1 보다는 명확한 user가 더 좋은 변수이름이지 않을까요?